authenticator.js

```diff
@@ -81,6 +82,13 @@ AuthenticateGithub.prototype.getAuthorizationToken = function(username, password
       if (err) reject(err);
       else resolve(res.token);
     });
+  }).then(this.githubOrg && function(token) {
+    return new Promise(function(resolve, reject) {
+      github.orgs.getMember({ user: username, org: _this.githubOrg }, function(err, res) {
```

Comment:
I have confirmed this behaviour on the live API, yes.
These two authenticating as me, a member of npm:
```
 ~ ⮀ curl -siLn https://api.github.com/orgs/npm/members/nathan7 | grep '^HTTP'     
HTTP/1.1 204 No Content
 ~ ⮀ curl -siLn https://api.github.com/orgs/npm/members/itzmjauz | grep '^HTTP'
HTTP/1.1 404 Not Found
```
These two unauthenticated:
```
 ~ ⮀ curl -siL https://api.github.com/orgs/npm/members/nathan7 | grep '^HTTP' 
HTTP/1.1 302 Found
HTTP/1.1 204 No Content
 ~ ⮀ curl -siL https://api.github.com/orgs/npm/members/itzmjauz | grep '^HTTP' 
HTTP/1.1 302 Found
HTTP/1.1 404 Not Found
```
The GitHub API client you're using handles the redirect — all "not a member" paths return a 404.